src/Hepteracts.ts

```diff
@@ -15,8 +15,10 @@ export interface IHepteractCraft {
     DISCOUNT?: number 
 }
 
-type hepteractTypes = 'chronos' | 'hyperrealism' | 'quark' | 'challenge' |
-                      'abyss' | 'accelerator' | 'acceleratorBoost' | 'multiplier'
+export type hepteractTypes = 'chronos' | 'hyperrealism' | 'quark' | 'challenge' |
+    'abyss' | 'accelerator' | 'acceleratorBoost' | 'multiplier'
+export const hepteractTypeList = ['chronos', 'hyperrealism', 'quark', 'challenge',
+    'abyss', 'accelerator', 'acceleratorBoost', 'multiplier' ]  //give me your forEach magic
```

Comment:
```suggestion
export const hepteractTypeList = ['chronos', 'hyperrealism', 'quark', 'challenge',
    'abyss', 'accelerator', 'acceleratorBoost', 'multiplier' ] as const;

export type hepteractTypes = typeof hepteractTypeList;
```

Reply:
I had to use `export type hepteractTypes = typeof hepteractTypeList[number];` otherwise there were tons of errors across the whole solution.